### ISSUE TITLE
tag_builder issue (action_view)

### ISSUE THREAD

**Issue by soumojit91** [patricklindsay/wice_grid#10]
**Error:** undefined local variable or method `tag_builder' for #<#<Class:0x007fab7a838d08>:0x007fab7e850850>

**Definition:** Trying to use wice_grid on rails 5. I have checked the lib locally, it does contain the patch by fabio for tag_builder. But I am still getting this error.

**Stack trace**
Completed 500 Internal Server Error in 18366ms (ActiveRecord: 5.5ms)

NameError - undefined local variable or method `tag_builder' for #<#<Class:0x007fab7a838d08>:0x007fab7e850850>:
   () Users/soumojit/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/bundler/gems/wice_grid-b6a2fefd4d10/lib/wice/wice_grid_core_ext.rb:119:in `public_tag_options'
   () Users/soumojit/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/bundler/gems/wice_grid-b6a2fefd4d10/lib/wice/helpers/wice_grid_view_helpers.rb:245:in `grid_html'
   () Users/soumojit/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/bundler/gems/wice_grid-b6a2fefd4d10/lib/wice/helpers/wice_grid_view_helpers.rb:124:in `define_grid'
   () Users/soumojit/.rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/bundler/gems/wice_grid-b6a2fefd4d10/lib/wice/helpers/wice_grid_view_helpers.rb:74:in `grid'
  app/views/admin/nach_banks/index.html.erb:19:in `_app_views_admin_nach_banks_index_html_erb___3851827470493547924_70187269392920'
  actionview (5.0.5) lib/action_view/template.rb:159:in `block in render'
  activesupport (5.0.5) lib/active_support/notifications.rb:164:in `block in instrument'
  activesupport (5.0.5) lib/active_support/notifications/instrumenter.rb:21:in `instrument'
  activesupport (5.0.5) lib/active_support/notifications.rb:164:in `instrument'
  actionview (5.0.5) lib/action_view/template.rb:354:in `instrument'
  actionview (5.0.5) lib/action_view/template.rb:157:in `render'

**Comment by IceColdAlpha** [comment]
Yep me too.  :(

**Comment by patricklindsay** [comment]
Looks like you're using Rails 5.0.5, are you able to upgrade to 5.1? This may solve the problem.

I think someone previously mentioned that there was an issue with 5.0.x

**Comment by JasonBarnabe** [comment]
➡️ ➡️ #1  ⬅️ ⬅️

**Comment by patricklindsay** [comment]
Fixed with new version `4.0.1` release.

```
gem 'wice_grid', git: 'https://github.com/patricklindsay/wice_grid.git', tag: 'v4.0.1'

```
